static/js/l10n/extracted_strings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
we might want to have separate files for these. perhaps one per "app", the first being places

Reply:
done